src/test/java/com/puppycrawl/tools/checkstyle/TreeWalkerTest.java

```diff
@@ -247,8 +239,8 @@ public void testWithCheckNotHavingTreeWalkerAsParent() throws Exception {
 
         try {
             final String[] expected = CommonUtil.EMPTY_STRING_ARRAY;
-            verify(createChecker(createTreeWalkerConfig(checkConfig)),
-                    File.createTempFile("junit", null, temporaryFolder).getPath(), expected);
+            execute(checkConfig,
```

Comment:
Sure !

Reply:
better to not anwer "Sure/I will", to avoid noise and notifications wihtout action, better to make comment "done" to let maintaier be notified and know that not it is maintainer part to respond.